docs/tsconfig.json

```diff
@@ -10,6 +10,13 @@
     { "path": "../lib" },
     { "path": "../templaters" }
   ],
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "**/*.jsx", "**/*.js"],
+  "include": [
+    "next-env.d.ts",
+    "docs-env.d.ts",
```

Comment:
Minor:

Individual files should be listed in [the `files` array](https://www.typescriptlang.org/tsconfig#files). `include` is meant for globs :smile: 

The benefit of using `files` is:

> An error occurs if any of the files can’t be found.